readme.md

```diff
@@ -16,11 +16,6 @@ We use a few tools to help with development. To install or update the required d
 
 `rake dependencies`
 
-#### CocoaPods
-
-Simplenote for iOS uses [CocoaPods](http://cocoapods.org/) to manage third party libraries.
-Third party libraries and resources managed by CocoaPods will be installed by the `rake dependencies` command above.
-
 #### Why isn't Sparkle fetched with CocoaPods
 
 At the time of writing, the version of Sparkle distributed via CocoaPods doesn't support Sandboxing, a feature that we value in Simplenote macOS.
```

Comment:
This section of the README might need a quick update.
 - Now that CocoaPods is no longer used in the project to begin with, it feels odd to still have a mention of it in this section about Sparkle out of nowhere.
 - This also raises the question if nowadays we couldn't fetch Sparkle via SwiftPM and an xcframework? 🤔